qt/python/mantidqt/dialogs/spectraselectordialog.py

```diff
@@ -100,12 +100,18 @@ def on_colorfill_clicked(self):
 
     def _check_number_of_plots(self, selection):
         index_length = len(selection.wksp_indices) if selection.wksp_indices else len(selection.spectra)
-        if selection.plot_type == SpectraSelection.Tiled and len(selection.workspaces) * index_length > 25:
+        number_of_lines_to_plot = len(selection.workspaces) * index_length
+        if selection.plot_type == SpectraSelection.Tiled and number_of_lines_to_plot > 25:
```

Comment:
Between 10 and 25 tiled plots you would get one warning, and >25 would be another. This seems like unusual behavior.

Reply:
Ur unusual behaviour